### DISCUSSION TITLE
Clicking the search icon on Ente Auth should immediately allow you to type

### DISCUSSION THREAD

**Discussion by chkpwd** [ente-io/ente#3153] [Enhancements]
To start searching, you have to click on the search box and begin typing.

<img width="923" alt="image" src="https://github.com/user-attachments/assets/cf7d6f4b-2f8d-44af-94e8-8774eb47c145">

**Comment by ua741** [comment]
Duplicate of https://github.com/ente-io/ente/discussions/3272
